tools/c7n_gcp/tests/test_query.py

```diff
@@ -35,6 +35,7 @@ def test_gcp_resource_metadata_asset_type():
         'ml-model',
         'notebook',
         'sourcerepo',
+        'spanner-instance-backup',
```

Comment:
please don't add to this list, if the resource is supported by asset inventory, just add the metadata to the resource type

[spanner.googleapis.com/Instance](https://cloud.google.com/spanner/docs/reference/rest/v1/projects.instances)

Reply:
Fixed. Please check